package.json

```diff
@@ -33,15 +34,27 @@
     "src/",
     "dist/"
   ],
+  "exports": {
+    ".": "./dist/index.css",
+    "./stylelint-config": "./src/stylelint/external-config.js"
+  },
```

Comment:
I think this could be considered a breaking change since it would break consumers that are importing a specific file, if there are any.